pkg/dist-node/index.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
```

Comment:
Is there a method to implement this that does not require maintain multiple identical copies of the source code in the source control? That seems like a lot to do in order to add this `export`/`import` feature.

Reply:
Yes, there are a tool that convert ES Module syntax (ESM) to Common.js and include type definitions for TypeScript.